### ISSUE TITLE
IllegalStateException when using 2 databases

### ISSUE THREAD

**Issue by anewton1998** [reactiverse/reactive-pg-client#158]
When transferring data from one database to another, I get an IllegalStateException

```
java.lang.IllegalStateException
	at io.reactiverse.pgclient.impl.SocketConnection.schedule(SocketConnection.java:173)
	at io.reactiverse.pgclient.impl.ConnectionPool$PooledConnection.schedule(ConnectionPool.java:81)
	at io.reactiverse.pgclient.impl.PgPreparedQueryImpl.execute(PgPreparedQueryImpl.java:76)
	at io.reactiverse.pgclient.impl.PgCursorStreamImpl$QueryCursor.checkPending(PgCursorStreamImpl.java:78)
	at io.reactiverse.pgclient.impl.PgCursorStreamImpl$QueryCursor.access$800(PgCursorStreamImpl.java:40)
	at io.reactiverse.pgclient.impl.PgCursorStreamImpl.resume(PgCursorStreamImpl.java:145)
	at com.rcode3.vpr.verticles.seed.DocTableSeed$start$1$1$3$1$1.handle(DocTableSeed.kt:107)
	at com.rcode3.vpr.verticles.seed.DocTableSeed$start$1$1$3$1$1.handle(DocTableSeed.kt:11)
...
```

A reproducer project is here:

https://github.com/anewton1998/vertx-pg-reproducer

Instructions for reproducing the issue are in the README.md

**Comment by vietj** [comment]
thanks for the accurate reproducer

**Comment by vietj** [comment]
@anewton1998 can you have a look, the bug was fixed and a 0.10.2-SNAPSHOT should be available in Sonatype OSS snapshot repo

**Comment by anewton1998** [comment]
@vietj confirmed, the issue is fixed. My rx-ified verticle runs to completion without issue. The verticle using the standard API has some sort of logic bug around closing the connection, but I believe that is a bug on my end somewhere.

Thanks a lot for taking care of this.
